src/main/java/org/commonjava/util/gateway/services/ProxyConstants.java

```diff
@@ -10,5 +10,5 @@
 
     // Auto generated, ignore such upstream headers
     public static final List<String> FORBIDDEN_HEADERS =
-                    Arrays.asList( "content-length", "connection", "transfer-encoding" );
+                    Arrays.asList( "connection", "transfer-encoding" );
```

Comment:
I filter content-length from upstream because it may break sth on client side. I do not remember what exactly it was. (maybe sth related to http2 where the content-length header is not expected). anyway, if we don't filter it, something will break.

Reply:
I thought there was something about Quarkus skimming it out...but I'm not sure what the problem was

Reply:
See below, I've reinstated that forbidden header.

Reply:
I see. btw, I just found the reason in the comment, "Raw content-length/connection header breaks http2 protocol."